config/config.yml

```diff
@@ -13,27 +16,27 @@ providers:
 permissions:
   # see full format details in 'permissions.cfg'
   # for each definition:
-  #   can refer to users and groups that will be created dynamically as needed with corresponding information
-  #   plain user or group name not matched against below entry will create it with defaults for other fields
+  #   can refer to below 'users' and 'groups' that will be created dynamically as needed with corresponding information
+  #   user or group name not matched against any extended definition will be created with defaults for other fields
 
 # extended definitions for users to create
 # default values will be used for missing optional fields
 #   not providing an entry referenced somewhere else (e.g.: in permissions)
 #   will default to only using the user name and defaults for every other field
 users:
   - username: username  # required if entry provided
-    password:           # optional
-    email:              # optional
-    group: <groupname>  # optional (either one of the groups, or plain string to generate it with defaults)
+    password:           # optional (default: random password of size MAGPIE_PASSWORD_MIN_LENGTH)
+    email:              # optional (default: <username>@mail.com)
+    group: <groupname>  # optional (one of 'groups' entry, or plain string to generate it with defaults, else anonymous)
```

Comment:
Reverse order : 'optional (default: "anonymous", one of ..)'

Why is "<groupname>"'s there but no "<email>"? Either remove or add missing placeholders.

Reply:
+1 for the reverse order

I don't understand the other comment about the `""`

Reply:
See the edit in previous comment